docs/index.rst

```diff
@@ -0,0 +1,260 @@
+.. toctree::
```

Comment:
This file is the main entry point for sphinx when the static site is regenerated.

The `.. toctree::` directive is how navigation is defined and dictates what other content is pulled in.